lyrics.py

```diff
@@ -0,0 +1,19 @@
+import re
+import sys
+
+from google import search
+from tswift import Song
+from tswift import SONG_RE
+
+
+def find_song(lyrics):
```

Comment:
This function should be a class method of `Song`. That way library users can benefit from `Song.find_song('lyrics')`.